static/js/components/dashboard/courses/StatusMessages.js

```diff
@@ -88,6 +88,31 @@ const messageForNotAttemptedExam = (course: Course) => {
   return message
 }
 
+const messageForAttemptedExams = (course: Course, passedExam: boolean) => {
+  const whenFailed = passedExam ? "" : "You did not pass the exam. "
+  const payAgain = course.has_to_pay
+    ? "If you want to re-take the exam, you need to pay again. "
+    : ""
+  let whenToSchedule = ""
+  if (course.can_schedule_exam) {
```

Comment:
why not

```
if (course.can_schedule_exam && !course.has_to_pay) {
  whenToSchedule = "Click above to reschedule an exam with Pearson."
}
```

Reply:
The rest of the `if else` statement should happen only based on `course.can_schedule_exam`